docs-next/pages/tutorials/getting-started-with-create-keystone-next-app/index.mdx

```diff
@@ -0,0 +1,134 @@
+It will generate a couple files for you and install all the dependencies you need to run the AdminUI and start using the GraphQL API.
+
+## Quick Start
+
+```sh
+yarn create keystone-next-app
+cd my-app
+yarn dev
+```
+
+## Installing your own Keystone instance
+
+Open your preferred shell and make sure you are in the folder you would like to create your new project in.
+`create-keystone-next-app` will generate a new folder with your new Keystone files in it.
+
+### Yarn
+
+Use [`yarn create`](https://classic.yarnpkg.com/en/docs/cli/create/):
+
+```sh
+cd your/path/
+yarn create keystone-next-app
+```
+
+### npm
+
+Use [`npm init`](https://docs.npmjs.com/cli/v7/commands/npm-init):
+
+```sh
+npm init keystone-next-app
+```
+
+_(`npm init <initializer>` is available in npm 6+)_
+
+### npx
+
+Use npms [`npx`](https://docs.npmjs.com/cli/v7/commands/npx):
+
+```sh
+cd your/path/
+npx create-keystone-next-app
+```
+
+_(npx comes with npm 5.2+)_
+
+The CLI will ask you to name your new app and for a way to connect to your database.
+Make sure you have your connection url ready for your database like e.g. `postgres://localhost/my-app` or `postgres://username:password@localhost/my-app`.
+
+_(If you run into trouble with the database, check out this guide on [Database Setup](https://www.keystonejs.com/quick-start/adapters))_
+
+## Opening your shiny new AdminUI
+
+<Image
+  src={require('./assets/adminui.png')}
+  alt="The AdminUI of Keystone showing the two lists: User and Posts"
+  width={1954}
+  height={764}
+/>
+
+After you ran `create-keystone-next-app` you can now `cd` into the folder that was created for you and run Keystone.
+
+```sh
+cd my-app
+yarn dev
+```
+
+This will generate the Admin UI pages via [NextJs](https://nextjs.org/) on [http://localhost:3000](http://localhost:3000).
+
+<Image
+  src={require('./assets/welcome-screen.png')}
+  alt="The welcome screen giving you the ability the create a new user to log into the AdminUI"
+  width={1328}
+  height={1020}
+/>
+
+When you visit the Admin UI for the first time you will be presented with a handy screen that asked you to create a user.
+Go ahead and create your first user.
+
+Now you can see your new Keystone Admin UI that comes with two visible lists and a bunch of files that were generated for you.
+
+## Output
+
+We now have a couple files in the newly generated folder. The two most important once you want to get into are `keystone.ts` and `schema.ts`.
+
+```sh
+.
+├── keystone.ts   # The main entry file for keystone
+├── node_modules  # Your dependencies
+├── package.json  # Your package.json with three scripts prepared for you
+├── schema.ts     # The schema of our data
+├── tsconfig.json # Your typescript config
+└── yarn.lock     # An finally your yarn lock file
+```
+
+## Scripts
+
+The files come with three npm scripts you can run locally.
+
+`dev`
+
+- The `start` script will run Keystone in production mode.
```

Comment:
```suggestion
- The `start` script will run Keystone in production mode.
```

This needs to be fixed.

Reply:
I'm not sure what needs fixing? The diff above seems to be empty?

Reply:
Oh I see now... it's the dev script